configs/minimal.yaml

```diff
@@ -176,7 +176,7 @@ BALANCE_PER_ADDITIONAL_CUSTODY_GROUP: 32000000000
 MAX_BLOBS_PER_BLOCK_FULU: 12
 MIN_EPOCHS_FOR_DATA_COLUMN_SIDECARS_REQUESTS: 4096
 
-# Whisk
+# EIP7441
 WHISK_EPOCHS_PER_SHUFFLING_PHASE: 4
 WHISK_PROPOSER_SELECTION_GAP: 1
```

Comment:
we might want to rename these to at least remove whisk?

Reply:
Yes, I had the same thought. I would like to remove "WHISK" from these variables. In your opinion, would it be appropriate to do it in this PR or should I do this in a follow up PR?

Reply:
We're in it now, i'd probably just do it here...